### PR TITLE
add com.github.jmlich.geotagging.json

### PR DESCRIPTION
**Pull request by jmlich**

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.github.jmlich.geotagging

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
Is this right way to trigger rebuild?

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
icons should be fixed now

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
command is fixed now

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
not sure if helped

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
command should be updated also in this repo

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
appstream-util validate

File: com.github.jmlich.geotagging.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  ],
+  "modules": [
+    {
+      "name": "exiv2",
+      "buildsystem": "cmake-ninja",
+      "builddir": true,
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/exiv2/exiv2"
```
Comment (bilelmoussaoui) [inline com.github.jmlich.geotagging.json:25]: This should be a tagged release or use a tarball if there's one.

File: com.github.jmlich.geotagging.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    {
+      "name": "geotagging",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/jmlich/geotagging.git"
```
Comment (bilelmoussaoui) [inline com.github.jmlich.geotagging.json:38]: Same

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
I have added git tag to each source.

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
I have forgot to git push --tags

**Comment by jmlich** [comment]
I am not sure if aarch64 build is required for flathub. It seems, that exiv2 issue was fixed in https://github.com/Exiv2/exiv2/commit/dd2d181755a6e642c0a8e3225ef5407fff49eb3a, which currently not available in tagged release.
We can use reference to commit instead of tag. Or we can wait for next release of exiv2 package.
Can you suggest best solution for the issue?

**Comment by nedrichards** [comment]
Use the commit for now. Or backport the fix as a patch if you feel super strongly about it.

**Comment by jmlich** [comment]
bot, build com.github.jmlich.geotagging
I don't want to maintain own patch to exiv2. I have used the latest and greatest commit. Now it will be hopefully working correctly.

**Pull request closed without merging**